fastlane_app.rb

```diff
@@ -1,13 +1,16 @@
 # External
 require "sinatra/base"
 
+set :server, "thin"
```

Comment:
I think we should have a comment about why we're using this

Reply:
Ah nice catch, this shouldn't be in here, gonna remove it! :)